src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -42,6 +42,8 @@ enum class StopCriteria { EARLY, HEURISTIC, NEVER };
  * @param include_stop_str_in_output if set to true stop string that matched generation will be included in generation output (default: false)
  * @param stop_token_ids vector of tokens that will cause pipeline to stop generating further tokens. Ignored for non continuous batching.
  * @param echo if set to true, output will include user prompt (default: false).
+ * @param logprobs number of top logprobs computed for each position, if set to 0, logprobs are not computed and value 0.0 is returned (default: 0).
+ *                 Currently only single top logprob can be returned, so any logprobs > 1 is treated as logprobs == 1. Default: 0.
```

Comment:
does actually logprob > 1 also imply that multiple tokens are returned? otherwise, just probabilities look useless